release/blocker_tracker_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+JIRA_TOKEN=""
+SLACK_WEB_HOOK_URL=""
+
+function main() {
+    local blocker_issues_keys=($(curl --request GET --url 'https://liferay.atlassian.net/rest/api/3/search?jql=project%20%3D%20%22PUBLIC%20-%20Liferay%20Product%20Delivery%22%20and%20labels%20%3D%20release-blocker%20and%20status%20%21%3D%20Closed&fields=issuekey' --user "jira-cloud-enterprisereleasehu@liferay.com:<$JIRA_TOKEN>" --header 'Accept: application/json' | jq -r '.issues[].key'))
```

Comment:
@kiwm, can you please move this code to be just before the `for blocker_issue_key in "${blocker_issues_keys[@]}"`?

Doing this, we just invoke the Jira API and declare this variable only when necessary. Besides, ideally, according to BChan's code convention, we should declare the variables just before their usage though there are some exceptions. :slightly_smiling_face: